askama/src/filters/alloc.rs

```diff
@@ -603,18 +788,6 @@ mod tests {
         assert_eq!(capitalize("ßß").unwrap().to_string(), "SSß".to_string());
     }
 
-    #[test]
-    fn test_title() {
-        assert_eq!(&title("").unwrap(), "");
-        assert_eq!(&title(" \n\t").unwrap(), " \n\t");
-        assert_eq!(&title("foo").unwrap(), "Foo");
-        assert_eq!(&title(" foo").unwrap(), " Foo");
-        assert_eq!(&title("foo bar").unwrap(), "Foo Bar");
-        assert_eq!(&title("foo  bar ").unwrap(), "Foo  Bar ");
-        assert_eq!(&title("fOO").unwrap(), "Foo");
-        assert_eq!(&title("fOo BaR").unwrap(), "Foo Bar");
```

Comment:
Also we would need to add a few cases to ensure that `foo\nbar` would render `Foo\nBar`. :)

Reply:
Right, added.